sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -331,6 +337,64 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
     console.printInfo(s"Spark master: $master, Application Id: $appId")
   }
 
+  override def processLine(line: String, allowInterrupting: Boolean): Int = {
```

Comment:
Could you write a comment above this line and explain the code is from org.apache.hadoop.hive.cli.CliDriver?